include/z64pause.h

```diff
@@ -0,0 +1,135 @@
+#ifndef Z64PAUSE_H
+#define Z64PAUSE_H
+
+#include "z64animation.h"
+#include "z64message.h"
+#include "z64view.h"
+
```

Comment:
```suggestion
#include "z64animation.h"
#include "z64message.h"
#include "z64ocarina.h"
#include "z64view.h"
```
Would also be worth adding `z64ocarina.h` in `z64message.h` as both these headers have an `OcarinaStaff*` struct member in them

Reply:
`OcarinaStaff` being only used for pointers I used a forward declaration of the struct instead